tools/cloudera/build_hue_cloudera.sh

```diff
@@ -46,6 +46,7 @@ function install_prerequisite() {
     export FORCEINSTALL=0
     export SQLITE3_PATH="$TOOLS_HOME/sqlite/bin/sqlite3"
     redhat7_ppc_install
+    # NODEJS install
     source /opt/rh/rh-nodejs14/enable
```

Comment:
not updated, still on 14

Reply:
Older Os'es (centos7, redhat7, ubuntu18) don't support node version 20. The latest they can support is node version 16, source : https://github.com/nodesource/distributions
cc: @JohanAhlen

Reply:
So to clarify after discussing with Johan, lets upgrade ALL node versions to 16 only so that we can be consistent across all installations.

Reply:
@JohanAhlen @athithyaaselvam 
Node 16 has actually already reached end of life and looking at EOL for the older Operating systems we are discussing here we see why there is no later node version for them:

- ubuntu18 End of Life April 30, 2023 (already dead)
- centos7 End of Life June 30, 2024
- redhat7 End of Maintenance June 30, 2024

Does it even make sense to support Hue on a linux version that has reached EOL?
I understand that ideally we should have the same node support across the installations, but forcing the whole project to use an EOL version in order to support linux versions that are dead or about to die doesn't make much sense either.

Reply:
So can we keep node version 16 for the older OS'es (centos7, redhat7, ubuntu18) and support node version 20 for the new operating systems?